backend/db/analysis.py

```diff
@@ -167,7 +167,7 @@ def counts_over_time(
         schema, local, from_cache=True)
 
     # Pivot
-    values = 'count' if method == 'counts_table' else 'delta'
+    values = 'counts' if method == 'counts_table' else 'delta'
```

Comment:
## Bugfix: Counts tables totals
There was a typo here. This is a general function but it has some split logic on counts vs deltas. However, in one place it was declaring the string `count` and later was checking for the string `counts`. The spellings were not equal, so that logic was never running.

Basically, this ended up manifesting a problem in which the tables for both counts and deltas were being treated the same, such that for both tables, the aggregated sum for that given day was what was being displayed. This is correct for deltas. However, for counts, we don't want to sum up all of the counts for all runs in a given day, we just want to check the counts for the latest run.

This bug caused no problem for deltas.